docs/build/get-started.md

```diff
@@ -548,8 +548,8 @@ mod tests {
         config_path = "<your path>/hello-world/.fluence/tmp/Config.toml",
         modules_dir = "<your path>/localdev/hello-world/target/wasm32-wasi/release"
     )]
-    fn test_hello_fluence(hw: marine_test_env::hello_world::ModuleInterface) { //3
-        let greeting = hw.hello_fluence();
+    fn test_hello_world(hw: marine_test_env::hello_world::ModuleInterface) { //3
+        let greeting = hw.hello_world();
         assert_eq!(greeting, "Hello, Fluence".to_string());
```

Comment:
looks like this needs to be hello world as well. boring.

Reply:
Nah, it's fine. Fluence is a world of its own, why not.